### PR TITLE
RAT-420: fixed checkstyle and removed unused methods in config package

### PR DESCRIPTION
**Pull request by Claudenw**
Fix for RAT-420

### CODE REVIEW

File: apache-rat-core/src/main/java/org/apache/rat/config/exclusion/ExclusionUtils.java [hunk 1]
```diff
@@ -280,7 +280,7 @@ public static Optional<MatchPatterns> notMatchPattern(final Iterator<String> ite
 
     /**
      * @param filename file name to cleanup.
-     * @return
+     * @return cleaned up file name.
```
Comment (ottlinger) [inline apache-rat-core/src/main/java/org/apache/rat/config/exclusion/ExclusionUtils.java:283]: @Claudenw the class has multiple unused methods - should we just remove them?
(same in other classes in this package/subpackage)

Comment (Claudenw) [reply apache-rat-core/src/main/java/org/apache/rat/config/exclusion/ExclusionUtils.java:283]: Removing methods always makes me nervous.  But I am going to say yes, remove them.

File: apache-rat-core/src/main/java/org/apache/rat/config/exclusion/ExclusionUtils.java [hunk 2]
```diff
@@ -209,110 +185,4 @@ private static void verifyFile(final File file) {
             throw new ConfigurationException(format("%s is not a valid file.", file));
         }
     }
-
```
Comment (ottlinger) [inline apache-rat-core/src/main/java/org/apache/rat/config/exclusion/ExclusionUtils.java:212]: @Claudenw I'm a bit suspicious if I removed too much ....
some of the methods are used by Reflection in other PRs and failed the build. But the build runs through .... is that code really unused or am I missing something here?

File: apache-rat-core/src/main/java/org/apache/rat/config/AddLicenseHeaders.java [hunk 3]
```diff
@@ -23,25 +23,10 @@
...
-    FALSE, 
-    /** Add headers and overwrite the old file */
-    FORCED;
-
-    private static final char SEPARATOR = '|';
-
-    /**
-     * @return pretty-printed list of all available options.
-     */
-    public static String getValuesForHelp() {
```
Comment (ottlinger) [inline apache-rat-core/src/main/java/org/apache/rat/config/AddLicenseHeaders.java:38]: @Claudenw does a call to this method miss somewhere in the utils module in order to print all available methods?

File: apache-rat-core/src/main/java/org/apache/rat/config/exclusion/ExclusionUtils.java (hunk 2 above)
Comment (Claudenw) [reply apache-rat-core/src/main/java/org/apache/rat/config/exclusion/ExclusionUtils.java:212]: I think it is fine.

**Pull request merged**